packages/plugin/src/modules/getRequestOptions.js

```diff
@@ -0,0 +1,32 @@
+import { getOptionsForRequest } from './options'
```

Comment:
`options.js` is still duplicated between app and plugin.
Perhaps out of scope of this PR, but we should remove that duplication too.
We could import it in the app from the plugin, although it feels it belongs more in the app than in the plugin, given the options UI etc...

Reply:
Agreed, but we should do that in a separate PR!